src/Snakemake/rules/QC/multiqc.smk

```diff
@@ -20,7 +20,7 @@ rule multiqcBatch:
     log:
         "logs/report/multiqc.log",
     singularity:
-        config["singularity"]["multiqc"]
+        config["singularity"].get("multiqc", config["singularity"].get("default", ""))
     shell:
```

Comment:
Why not use wrapper?

Reply:
No particular reason, maby because it does a copy afterwards ...

Reply:
You should be able to specify output, at least according to the wrapper

https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/multiqc.html 

could be a possible enhancement and a good task for Linköping